lib/pronto/formatter/github_status_formatter.rb

```diff
@@ -0,0 +1,85 @@
+require_relative 'github_status_formatter/sentence'
+
+module Pronto
+  module Formatter
+    class GithubStatusFormatter
+      def initialize(opts = {})
+        @level_mapping ||= opts.fetch(:level_mapping, {})
```

Comment:
The reason for implementing this was that Reek was returning only `warning`s even though stuff like `Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)` sound pretty subjective to me. Now when I think about it, it'd probably be better to address the issue at mmozuras/pronto-reek than here.

I'll get rid of `opts` parameter for now.